modules/api/src/user/entities/user.entity.ts

```diff
@@ -0,0 +1,27 @@
+import { Column, Entity, Index, PrimaryGeneratedColumn } from 'typeorm';
+
+@Entity()
+export class User {
```

Comment:
we should make a base IUser class that the INewUser and IPassportUser inherit from since they all need the same base properties.

Reply:
apparently breaks passport, so that's cool.